#set start
start_x: 1.0
start_y: 1.0
start_yaw: 0.0
#set goal 
goal_x: 6.0
goal_y: 10.0
goal_yaw: 0.0
#set parameter of robot
v_min: 0.0
v_max: 5.0
v_reso: 0.1
w_min: 0.0
w_max: 3.0
w_reso: 0.05
v_acc: 1.0
w_acc: 0.5
#set initial position
init_x: 0.0
init_y: 0.0
init_yaw: 0.0


